code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1626,9 +1630,6 @@
 	if(istype(head, /obj/item/clothing/head/wizard) || istype(head, /obj/item/clothing/head/helmet/space/rig/wizard))
 		threatcount += 2
 
-	//Check for nonhuman scum
-	if(dna && dna.mutantrace && dna.mutantrace != "none")

```

Comment:
we originally had this, but it ultimately results in beepsters wanting to arrest Tajarans and such for no reason or really trivial stuff--considering races are very much a thing on Para (and not TG, where this came from), this might be a case where it's ok this dies.